receiver/statsdreceiver/protocol/metric_translator.go

```diff
@@ -98,6 +100,55 @@ func buildSummaryMetric(desc statsDMetricDescription, summary summaryMetric, sta
 	}
 }
 
+func buildHistogramMetric(desc statsDMetricDescription, histogram histogramMetric, startTime, timeNow time.Time, ilm pmetric.ScopeMetrics) {
+	nm := ilm.Metrics().AppendEmpty()
+	nm.SetName(desc.name)
+	expo := nm.SetEmptyExponentialHistogram()
+	expo.SetAggregationTemporality(pmetric.MetricAggregationTemporalityDelta)
+
+	dp := expo.DataPoints().AppendEmpty()
+	agg := histogram.agg
+
+	dp.SetCount(agg.Count())
+	dp.SetSum(agg.Sum())
```

Comment:
Does goexpohisto support min/max? (I assume it does not, but just double checking in case we can add it)

Reply:
yes, it does. this PR dates to a time before the OTLP protocol had added it in the collector (due to the Gogo block), oops! Fixed (with new test apparatus).